packages/app-page-builder/src/editor/components/Element.tsx

```diff
@@ -115,7 +128,7 @@ const Element = (props: ElementProps) => {
                         <Draggable target={plugin.target} beginDrag={beginDrag} endDrag={endDrag}>
                             {renderDraggable}
                         </Draggable>
-                        {plugin.render({ element })}
+                        {renderedPlugin}
```

Comment:
Then here you can simply call that function:
```js
{renderPlugin({ plugin, element })}
```

It's easier to read code and navigate around function calls this way.

Reply:
yep